js/generation-form.js

```diff
@@ -0,0 +1,69 @@
+
+const type = {
+  palace : 'Дворец',
+  flat : 'Квартира',
+  bungalow : 'Бунгало',
+  house : 'Дом',
+  hotel : 'Отель',
+};
+// Объект данных
+const offerList = document.querySelector('#map-canvas');
+const  generate = (Item)=> {
```

Comment:
```suggestion
const  generate = (item)=> {
```
просто так, с большой буквы в JS ничего не пишут)